pymc/tests/test_aesaraf.py

```diff
@@ -529,3 +529,23 @@ def test_rvs_to_value_vars():
     assert a_value_var in res_ancestors
     assert b_value_var in res_ancestors
     assert c_value_var in res_ancestors
+
+
+def test_rvs_to_value_vars_nested_transform_bug():
```

Comment:
It's better to name a test after the things being tested, instead of a reference to a previous issue/bug that likely won't hold up over time.  In general, a developer seeing this in the future shouldn't need to know/read old issues in order to understand what's being tested; this applies to test names, as well.